zero_bin/leader/src/main.rs

```diff
@@ -141,3 +141,18 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+
+/// Attempt to load in the local `.env` if present and set any environment
+/// variables specified inside of it.
+///
+/// To keep things simple, any IO error we will treat as the file not existing
+/// and continue moving on without the `env` variables set.
+fn load_dotenvy_vars_if_present() {
+    match dotenv() {
+        Ok(_) | Err(dotenvy::Error::Io(io::Error { .. })) => (),
+        Err(e) => warn!(
+            "Found local `.env` file but was unable to parse it! (err: {})",
+            e.to_string()
```

Comment:
Nitpick
```suggestion
            "Found local `.env` file but was unable to parse it! (err: {e})"
```

Reply:
Ah, I hadn't seen the auto-squash.. I guess it's fine for this comment?

Reply:
Sorry about the auto-squash! I'll not set that anymore unless I know there are no more requested changes.

I'm surprised that `clippy` didn't pick this up... I can do a `clippy` pass PR right now on the repo and manually include this change.